src/api/services/todos.service.js

```diff
@@ -62,3 +62,17 @@ exports.deleteTodo = async (todoId, dateObject) => {
   todo.addedInCalendar.delete(date);
   await todo.save();
 };
+
+exports.modifyTodoCheckButton = async (id, isComplete, date) => {
+  const todo = await Todo.findById(id).exec();
+
+  if (!todo || !todo.addedInCalendar.has(date)) {
+    return {
+      isSuccess: false,
+    };
+  }
+
+  todo.addedInCalendar.set(date, { isComplete });
```

Comment:
PR에 논의사항으로 남겨주신 부분은 이 코드 문제일 것 같아요.
map.set(key, value)를 사용하면 해당 key에 대한 value가 통으로 바뀌게 되는데, 스키마에서 memo의 default value를 빈 스트링으로 주고 있기 때문에 함께 들어가는 것으로 보입니다.

JS map 객체라고 가정하면 아래와 같이 처리가 가능한데요, 한 번 테스트 해봐주시겠어요? 마지막에 true로 할당해주는 부분은 패러미터를 고려해서 수정하면 될 것 같고요.

```suggestion
  todo.addedInCalendar.get(date).isComplete = true;
```